App/src/pages/CourseDetails.tsx

```diff
@@ -94,9 +94,9 @@ export default function CourseDetails() {
                     width="240px"
                     height="40px"
                     margin="auto"
-                    onClick={() => navigate(BaseRoutes.CourseBrowser)}
+                    onClick={() => navigate(-1)}
```

Comment:
This is actually a really nice change, it'd be useful in a lot of spots throughout the frontend, nice work thinking of it.